python/lsst/daf/butler/core/registry.py

```diff
@@ -104,73 +107,74 @@ def addDataset(self, ref, uri, components, run, producer=None):
 
         Parameters
         ----------
-        ref: `DatasetRef`
+        ref : `DatasetRef`
             Identifies the `Dataset` and contains its `DatasetType`.
-        uri: `str`
+        uri : `str`
             The URI that has been associated with the `Dataset` by a
             `Datastore`.
-        components: `dict`
+        components : `dict`
             If the `Dataset` is a composite, a ``{name : URI}`` dictionary of
             its named components and storage locations.
-        run: `Run`
+        run : `Run`
             The `Run` instance that produced the Dataset.  Ignored if
             ``producer`` is passed (`producer.run` is then used instead).
             A Run must be provided by one of the two arguments.
-        producer: `Quantum`
+        producer : `Quantum`
             Unit of work that produced the Dataset.  May be ``None`` to store
             no provenance information, but if present the `Quantum` must
             already have been added to the Registry.
 
         Returns
         -------
-        ref: `DatasetRef`
+        ref : `DatasetRef`
             A newly-created `DatasetRef` instance.
 
         Raises
         ------
-        e: `Exception`
+        Exception
             If a `Dataset` with the given `DatasetRef` already exists in the
             given Collection.
         """
         raise NotImplementedError("Must be implemented by subclass")
 
     @abstractmethod
     def associate(self, collection, refs):
-        """Add existing `Dataset`s to a Collection, possibly creating the
+        """Add existing `Dataset`\ s to a Collection, possibly creating the
         Collection in the process.
 
         Parameters
         ----------
-        collection: `str`
-            Indicates the Collection the `Dataset`s should be associated with.
-        refs: `[DatasetRef]`
+        collection : `str`
+            Indicates the Collection the `Dataset`\ s should be associated with.
+        refs : `[DatasetRef]`
```

Comment:
`list of `DatasetRef`?

Reply:
Yes, I thought I had found them all but my regex clearly wasn't quite right. I'll have another pass.